modules/apps/user/user-test/src/testIntegration/java/com/liferay/user/service/test/UserSetDigestTest.java

```diff
@@ -93,9 +95,9 @@ private void _testAddUserWithWorkflowHelper(
 
 		long creatorUserId = 0;
 
-		_company = CompanyTestUtil.addCompany();
+		Company company = _companyLocalService.getCompany(TestPropsValues.getCompanyId());
 
-		long companyId = _company.getCompanyId();
+		long companyId = company.getCompanyId();
```

Comment:
probably we can use directly `TestPropsValues.getCompanyId()` instead of getting the company and the companyId